include/cxx_project.sh

```diff
@@ -59,8 +59,26 @@ function make_project
 
 	DATE=`date "+date: %Y-%m-%d"` 
 	TIME=`date "+time: %H:%M:%S"`
+	
 	# filesystem creation
-	mkdir $PROJECT_NAME $PROJECT_NAME/include $PROJECT_NAME/source $PROJECT_NAME/build
+	if [ ! -d "$PROJECT_NAME" ]; 
+	then
+		mkdir "$PROJECT_NAME"
+	else
+		if [ "$4" == "true" ]
+		then
+			rm -rf "$PROJECT_NAME"
```

Comment:
I suggest 
rm -rf "$PROJECT_NAME/*"
so you do not need to create the directory at line 72.